src/components/ai-icon/ai-avatar.tsx

```
@@ -0,0 +1,42 @@
+    /**
+     * The text to show to screenreaders and other assistive tech.
+     */
+    @Prop({ reflect: true })
+    public label: string;
+
+    @Prop({ reflect: true })
+    public static: false;
+
+    public render() {
+        return [
+            <div slot="trigger">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)